scorescreen.py

```diff
@@ -106,13 +104,20 @@ def run(self, game, scores):
             game.screen.fill(BLACK)
             game.screen.blit(self.scorescreen, (350, 0))
 
-            msg = self.font3.render(_("GAME OVER"), 2, WHITE)
+            msg = self.font3.render(
```

Comment:
Why was this changed?  Vertical space in IDE becomes more of an issue.

Reply:
Changed!

Reply:
Still there.

Reply:
`msg = self.font3.render(_("GAME OVER"), 2, WHITE)`
This one are we talking about?

Reply:
I was speaking of your pull request, not the code line where I pressed the button.  :grin:

Reply:
Oh really sorry for misunderstanding, I continued from #16 all the changes were recommended there and then fixed by the author, but if you want me to do the changes here I will do them because there aren't much of them, if you can review/mark at the lines where there is vertical space you want changes.
Thanks

Reply:
Okay, was not aware.  Have now expanded outdated comments on the older pull request.  I withdraw my suggestion.  I'll wait for @chimosky to review.